lib/win32/certstore/mixin/helper.rb

```diff
@@ -23,10 +23,10 @@ module Mixin
       module Helper
 
         # PSCommand to search certificate from thumbprint and convert in pem
-        def cert_ps_cmd(thumbprint, store_name)
+        def cert_ps_cmd(thumbprint, store_name, store_location)
```

Comment:
same here `cert_ps_cmd(thumbprint, store_name, store_location: CERT_SYSTEM_STORE_LOCAL_MACHINE)` and modify the callers